dump/dash_util.py

```diff
@@ -0,0 +1,96 @@
+import base64
+import uuid
+import socket
+import ipaddress
+from google.protobuf.message import Message
+from dash_api.appliance_pb2 import *
+from dash_api.vnet_pb2 import *
+from dash_api.eni_pb2 import *
+from dash_api.acl_group_pb2 import *
+from dash_api.acl_rule_pb2 import *
+from dash_api.acl_in_pb2 import *
+from dash_api.acl_out_pb2 import *
+from dash_api.prefix_tag_pb2 import *
+from dash_api.types_pb2 import *
```

Comment:
For importing the protobuf objects need further [need further processing](https://github.com/sonic-net/sonic-utilities/blob/a23a880d7589f09fe23b8baa5e38445a8531524f/dump/dash_util.py#L51)

Reply:
Is it possible to only `from dash_api.types_pb2 import *`?
Because these objects are both defined in the https://github.com/sonic-net/sonic-dash-api/blob/master/proto/types.proto

Reply:
All the modules are required since we are performing [type comparison with the existing types](https://github.com/sonic-net/sonic-utilities/blob/1cca178572f82fd88d2e9cedeb707e997e3a1480/dump/dash_util.py#L63) since type comparison fails otherwise (Example where isinstance fails even though type is same):
`obj = <class 'types_pb2.Guid'> and decode_types = (<class 'types_pb2.IpAddress'>, <class 'types_pb2.Guid'>, <class 'types_pb2.IpPrefix'>) isinstance(obj, decode_types) = False`
Will do further analysis and check if anything can be changed

Reply:
Resolved by comparing the module name and class name instead of `isinstance`